### ISSUE TITLE
[JENKINS-69276] No focus state on signin page checkbox (regression in 2.361)

### ISSUE THREAD

**Issue by timja-bot** [timja-org/jenkins-gh-issues-poc-11-04#6567]
<p>Sign in page has lost it's focus state for the keep me signed in checkbox</p>

<p> <span class="image-wrap" style=""><a id="58712_thumb" href="https://issues.jenkins.io/secure/attachment/58712/58712_image-2022-08-08-08-49-04-881.png" title="image-2022-08-08-08-49-04-881.png" file-preview-type="image" file-preview-id="58712" file-preview-title="image-2022-08-08-08-49-04-881.png"><img src="https://issues.jenkins.io/secure/thumbnail/58712/_thumb_58712.png" style="border: 0px solid black" role="presentation"/></a></span> </p>

---
<details><summary><i>Originally reported by <a title="Tim Jacomb" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=timja">timja</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-69276" target="_blank">No focus state on signin page checkbox (regression in 2.361)</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="frankie699" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=frankie699">frankie699</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2022-08-17T15:32:20+00:00
<li><b>imported</b>: 2025-11-05
</ul></i>
</details>

**Comment by timja-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck">danielbeck</a>:</i>
<p>Works in 2.360, broken in 2.361.</p>
